pkg/operator/ceph/csi/spec.go

```diff
@@ -196,7 +201,7 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 	tp.RBDGRPCMetricsPort = getPortFromENV("CSI_RBD_GRPC_METRICS_PORT", DefaultRBDGRPCMerticsPort)
 	tp.RBDLivenessMetricsPort = getPortFromENV("CSI_RBD_LIVENESS_METRICS_PORT", DefaultRBDLivenessMerticsPort)
 
-	if ver.Minor < provDeploymentSuppVersion {
+	if ver.Major > KubeMinMajor || ver.Minor < provDeploymentSuppVersion {
```

Comment:
the version check may need an update like mentioned below

Reply:
done